concourse/scripts/pipelines-bosh-cloudfoundry.sh

```diff
@@ -8,7 +8,8 @@ export TARGET_CONCOURSE=deployer
 $("${SCRIPT_DIR}/environment.sh" "$@")
 "${SCRIPT_DIR}/fly_sync_and_login.sh"
 
-env=${DEPLOY_ENV}
+env="${DEPLOY_ENV}"
+pipelines_to_update="${PIPELINES_TO_UPDATE:-create-bosh-cloudfoundry destroy-microbosh destroy-cloudfoundry autodelete-cloudfoundry failure-testing}"
```

Comment:
I think `destroy-cloudfoundry` should come before `destroy-microbosh` as that's the order they should be run in.